README.md

```diff
@@ -800,6 +800,12 @@ All marks support the following optional channels:
 
 The **fill**, **fillOpacity**, **stroke**, **strokeWidth**, **strokeOpacity**, and **opacity** options can be specified as either channels or constants. When the fill or stroke is specified as a function or array, it is interpreted as a channel; when the fill or stroke is specified as a string, it is interpreted as a constant if a valid CSS color and otherwise it is interpreted as a column name for a channel. Similarly when the fill opacity, stroke opacity, object opacity, stroke width, or radius is specified as a number, it is interpreted as a constant; otherwise it is interpreted as a channel.
 
+Every variable color channel is scaled with the *color* scale unless its defined values are all valid CSS color strings, in which case they are used directly. This is equivalent to defining the **fill** or **stroke** channel as an object with a *value* and an undefined *scale* property. To opt out of automatic detection, specify the scale like so:
+```javascript
+Plot.dot(data, {stroke: {value: "fieldName", scale: "color"}}) // use the color scale
+Plot.dot(data, {stroke: {value: "fieldName", scale: null}})    // directly use the given values
```

Comment:
```suggestion

```js
Plot.dot(data, {stroke: {value: "fieldName", scale: "color"}}) // use the color scale
Plot.dot(data, {stroke: {value: "fieldName", scale: null}})    // directly use the given values
```